From 44e98653ae16f158fb50794976550d9b768e6979 Mon Sep 17 00:00:00 2001
From: Anthony <18110175@student.hcmute.edu.vn>
Date: Wed, 8 Mar 2023 15:06:01 +0700
Subject: [PATCH 19/22] Update Abc14.md 2

---
 Abc14.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Abc14.md b/Abc14.md
index caed834..2250384 100644
--- a/Abc14.md
+++ b/Abc14.md
@@ -22,4 +22,4 @@ README.md
 README.md
 README.md
 README.md
-README.md 1
+README.md 1 2
-- 
2.32.0 (Apple Git-132)

