From 5b05ace963d55a6311f37c17e63a29b594b63575 Mon Sep 17 00:00:00 2001
From: Anthony <18110175@student.hcmute.edu.vn>
Date: Thu, 9 Mar 2023 23:43:27 +0700
Subject: [PATCH 20/22] Update Abc14.md

---
 Abc14.md | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Abc14.md b/Abc14.md
index 2250384..acbb4f1 100644
--- a/Abc14.md
+++ b/Abc14.md
@@ -23,3 +23,5 @@ README.md
 README.md
 README.md
 README.md 1 2
+
+In Main
-- 
2.32.0 (Apple Git-132)

